src/layer/BingTiledImageLayer.js

```diff
@@ -80,6 +88,18 @@ define([
             return 256 << (levelNumber + 1);
         };
 
+        BingTiledImageLayer.prototype.createBingLogotype = function () {
+            // Locate Bing logo in the upper left corner
+            var offset = new Offset(WorldWind.OFFSET_FRACTION, 1, WorldWind.OFFSET_FRACTION, 0);
```

Comment:
Let's place the Bing logo where it appears now. There's some amount of pixel inset, right?